Dalamud/Game/Command/CommandManager.cs

```diff
@@ -41,12 +41,31 @@ private CommandManager(Dalamud dalamud)
             (nint)ShellCommands.MemberFunctionPointers.TryInvokeDebugCommand,
             this.OnTryInvokeDebugCommand);
         this.tryInvokeDebugCommandHook.Enable();
-
-        this.console.Invoke += this.ConsoleOnInvoke;
     }
 
     /// <inheritdoc/>
-    public ReadOnlyDictionary<string, IReadOnlyCommandInfo> Commands => new(this.commandMap);
+    [Api13ToDo("Make this sensible. Don't use exposed API for internal structures.")]
```

Comment:
API note: some people can/do use this, we may need to create an interface to expose commands. (See: SimpleTweaks)

Reply:
Yeah, we definitely still want this. The intent isn't to remove it, but to replace it with something nicer.